app/javascript/components/ModalWizard/ModalWizardStateProvider.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { bindMethods } from '../../common/helpers';
+
+class ModalWizardStateProvider extends React.Component {
+  constructor() {
+    super();
+    this.state = { activeStepIndex: 0 };
+    bindMethods(this, ['prevStep', 'nextStep']);
+  }
+
+  prevStep() {
+    const { activeStepIndex } = this.state;
+    this.setState({ activeStepIndex: Math.max(activeStepIndex - 1, 0) });
```

Comment:
i think `activeStepIndex` can be lifted to Redux... let's go with this for now though! I will revisit it some in spare time soon and open a subsequent PR. This will keep us iterating fast for now and enable development of the other steps.

Reply:
Yep! The reason I contained the state in a small `StateProvider` component is so it could be easily replaced later.